src/libcore/ops.rs

```diff
@@ -470,26 +470,37 @@ rem_impl_float! { f32 f64 }
 ///
 /// # Examples
 ///
-/// A trivial implementation of `Neg`. When `-Foo` happens, it ends up calling
-/// `neg`, and therefore, `main` prints `Negating!`.
+/// An implementation of `Neg` for `Sign`, which allows the use of `-` to
```

Comment:
I stand by the necessity of this comma (Strunk & White II.3), but I certainly won't fight to the death over it.